src/components/CategoryFilter.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const categories = ['All', ...['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi']];
```

Comment:
- [ ] [OPTIONAL] Since you are defining the categories already with the "All" option inside it there's no need to use the spread operator, you can simply add it as a normal item of the array.